### PR TITLE
Cura 5943 ui usb printing

### PR DESCRIPTION
**Pull request by nallath**

### CODE REVIEW

File: plugins/MonitorStage/MonitorMainView.qml [hunk 1]
```diff
@@ -1,45 +1,61 @@
...
+
+    Loader
+    {
+        id: monitorViewComponent
+
+        anchors.top: parent.top
+        anchors.bottom: parent.bottom
+        anchors.left: parent.left
+
+        width: parent.width * 0.7
```
Comment (nallath) [inline plugins/MonitorStage/MonitorMainView.qml]: Needs rounding.

Comment (nallath) [reply plugins/MonitorStage/MonitorMainView.qml]: Actually; This should be at full width if the sidebar is not set at all.

File: resources/qml/PrinterOutput/MonitorButtonStyle.qml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import QtQuick.Controls.Styles 1.4
+import QtQuick.Layouts 1.3
+
+import UM 1.2 as UM
+import Cura 1.0 as Cura
+
+
+Component
+{
+    ButtonStyle
```
Comment (nallath) [inline resources/qml/PrinterOutput/MonitorButtonStyle.qml]: Styles should be defined in the theme. Also have a look if it's possible to ensure that we don't need a new unique style for these buttons.

File: plugins/MonitorStage/MonitorMainView.qml (hunk 1 above)
Comment (LipuFei) [reply plugins/MonitorStage/MonitorMainView.qml]: Done in 845b320

File: resources/qml/PrinterOutput/MonitorButtonStyle.qml (hunk 2 above)
Comment (LipuFei) [reply resources/qml/PrinterOutput/MonitorButtonStyle.qml]: Done in 6bb010e

**Pull request merged**